### ISSUE TITLE
Ability to set a default playlist

### ISSUE THREAD

**Issue by tim1mw** [tim1mw/pycon_capture#7]
I gather this needs a further API call...

All videos are added to playlists which are based on the room and day on which the video was filmed so people can find stuff based on the conference schedule. Ideally the playlist should also be put in chronological order.

**Comment by IanTaylorFB** [comment]
Yes, I'll post my thoughts about this later today, but you're right - separate api call.
